opus/application/static_media/js/browse.js

```diff
@@ -796,6 +725,134 @@ var o_browse = {
         }
     },
 
+    getCurrentStartObs: function(selector, tab, viewNamespace, galleryBoundingRect, browserResized) {
+        /**
+         * get the current startObs (for slider number) and the top item's
+         * obsNum in table view, this item will also be at the top row in
+         * gallery view (for setting scrollbar location)
+         */
+
+        // this will get the top left obsNum for gallery view or the top obsNum for table view
+        let firstCachedObs = $(selector).first().data("obs");
+        let firstCachedObsTop = $(selector).first().offset().top;
+        let calculatedFirstObs = (o_utils.floor((firstCachedObs - 1)/galleryBoundingRect.x) *
+                                  galleryBoundingRect.x + 1);
+        // For gallery view, the topBoxBoundary is the top of .gallery-contents
+
+        // For table view, we will set the topBoxBoundary to be the bottom of thead
+        // (account for height of thead)
+        let topBoxBoundary = (o_browse.isGalleryView() ?
+                          $(`${tab} .gallery-contents`).offset().top :
+                          $(`${tab} .gallery-contents`).offset().top + $(`${tab} .op-data-table thead th`).outerHeight());
+
+        // table: obsNum = calculatedFirstObs + number of row
+        // gallery: obsNum = calculatedFirstObs + number of row * number of obs in a row
+        // Note: in table view, we divide by the 2nd table tr's height because in some corner
+        // cases, the first table tr's height will be 1px larger than rest of tr, and this
+        // will mess up the calculation.
+        let tableRowHeight = $(`${tab} tbody tr`).eq(1).outerHeight();
```

Comment:
So what happens here if there's only one row in the table and we ask for info on the second row? OPUS continues to work, probably because the slider ends up being disabled anyway, but I'm guessing it returns "Undefined" here and then the math below returns something bogus.